File: Day 92/classwork/cw3.py
# 3) https://www.codewars.com/kata/57eeb8cc5f79f6465a0015c1/train/python

def is_kiss(words):
    words_list = words.split()
    for i in words_list:
        if len(i) > len(words_list):
            return "Keep It Simple Stupid"
    
    return "Good work Joe!"

File: Day 157/classwork/cw3.js
// 3) https://www.codewars.com/kata/57a1fd2ce298a731b20006a4/train/javascript


function isPalindrome(x) {
    if(x.toLowerCase() === x.split("").reverse().join("").toLowerCase()){
        return true
    }else{
        return false
    }
}